build/.vsts-pr.yml

```diff
@@ -18,3 +20,38 @@ jobs:
   - template: build.yml
     parameters:
       packageArtifacts: false
+
+- job: Docker
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: DockerCompose@0
+    displayName: Container registry login
+    inputs:
+      containerregistrytype: Azure Container Registry
+      azureSubscriptionEndpoint: $(azureSubscriptionEndpoint)
+      azureContainerRegistry: $(azureContainerRegistry)
+
+  - task: DockerCompose@0
+    displayName: Build FHIR Server Image
+    inputs:
+      action: Build services
+      dockerComposeFile: samples/docker/docker-compose.yaml 
+      projectName: fhirserverforazure
+      qualifyImageNames: true
+      additionalImageTags: |
+        pr$(system.pullRequest.pullRequestNumber)
+        $(Build.BuildId)
+
+  - task: DockerCompose@0
+    displayName: Push FHIR Server Image
+    inputs:
+      action: Push services
+      azureSubscriptionEndpoint: $(azureSubscriptionEndpoint)
+      azureContainerRegistry: $(azureContainerRegistry)
+      dockerComposeFile: docker-compose.yml
+      projectName: fhirserverforazure
+      qualifyImageNames: true
+      additionalImageTags: |
+        pr$(system.pullRequest.pullRequestNumber)
+        $(Build.BuildId)
```

Comment:
You might want to define these as variables. Also do we need 'latest' tag too?

Reply:
The latest tag is provided automatically. Tags changed to input parameters.